README.md

```diff
@@ -1,14 +1,18 @@
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 Creates an AWS Lambda function to clean up manual RDS snapshots
 on a scheduled interval using [truss-aws-tools](https://github.com/trussworks/truss-aws-tools).
 
 Creates the following resources:
 
-* IAM role for Lambda function find and delete expired RDS snapshots for a
+  IAM role for Lambda function find and delete expired RDS snapshots for a
```

Comment:
Was this intentional?

Reply:
...I was testing you, and you passed (but my code clearly did not).